app/javascript/main/index.js

```diff
@@ -18,17 +10,12 @@ WebpackerReact.setup({ MainApp })
 // For now we need to assign our view components to a const
 // In future iterations, this might not be the case
 
-const TodoShow = views.TodoShow
 
 /*
  * Build the application
  */
 function MainApp () {
```

Comment:
Alternatively, you could just have `WebpackerReact.setup({ MainApp: Routes })`

Reply:
That line doesn't define `MainApp`, it just tells WebpackerReact that it can refer to `Routes` as `MainApp` from the server side. We can then do all the subrouting in `Routes`.